modules/appnexusBidAdapter.js

```diff
@@ -338,10 +340,18 @@ function newBid(serverBid, rtbBid, bidderRequest) {
       rating: nativeAd.rating,
       sponsoredBy: nativeAd.sponsored,
       privacyLink: nativeAd.privacy_link,
+      address: nativeAd.address,
+      downloads: nativeAd.downloads,
+      likes: nativeAd.likes,
+      phone: nativeAd.phone,
+      price: nativeAd.price,
+      sale: nativeAd.saleprice,
       clickUrl: nativeAd.link.url,
+      clickLabel: nativeAd.displayurl,
       clickTrackers: nativeAd.link.click_trackers,
       impressionTrackers: nativeAd.impression_trackers,
       javascriptTrackers: nativeAd.javascript_trackers,
+      video: nativeAd.video.content
```

Comment:
need to add a check to verify the `nativeAd.video` property exists before reading the `content` field.  This otherwise throws an `undefined` error (such as in the case of a standard native image ad).

the check could just be `video: nativeAd.video && nativeAd.video.content`

Reply:
Thanks @jsnellbaker -- per internal conversations in slack i've dropped the video parameter entirely